packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -259,6 +259,45 @@ void main() {
     });
   });
 
+  testWidgets('picker honours showDayOfWeek supported only in date mode', (WidgetTester tester) async {
+    // This should work: showDayOfWeek is used in date mode
```

Comment:
Not sure if this comment and the one below are necessary since the description is pretty clear.

Reply:
You're right, I've removed it!